tests/D2L.CodeStyle.Analyzers.Test/Specs/ImmutabilityAnalyzer.cs

```diff
@@ -38,8 +39,16 @@ public enum Because {
 	}
 }
 
+#if NETFRAMEWORK
+namespace System.Runtime.CompilerServices {
+	// Polyfill for .NET framework
+    internal static class IsExternalInit {}
+}
+#endif

```

Comment:
We were using `init;` but our tests DLL is .NET Framework -- this is a polyfill.